app/views/directives/resource-service-bindings.html

```diff
@@ -0,0 +1,28 @@
+<div class="expanded-section mar-bottom-xl" ng-if="$ctrl.showBindings">
```

Comment:
Do we want `expanded-section` here? I believe it's currently an overview-specific style.